pkg/codegen/internal/test/testdata/simple-plain-schema-with-root-package/schema.json

```diff
@@ -0,0 +1,113 @@
+{
+  "version": "0.0.1",
+  "name": "example",
+  "types": {
+    "example::Foo": {
+      "properties": {
+        "a": {
+          "type": "boolean"
+        },
+        "b": {
+          "type": "boolean"
+        },
+        "c": {
+          "type": "integer"
+        },
+        "d": {
+          "type": "integer"
+        },
+        "e": {
+          "type": "string"
+        },
+        "f": {
+          "type": "string"
+        }
+      },
+      "required": ["a", "c", "e"],
+      "plain": ["a", "b", "c", "d", "e", "f"],
+      "type": "object"
+    }
+  },
+  "resources": {
+    "example::Component": {
+      "isComponent": true,
+      "properties": {
+        "a": {
+          "type": "boolean"
+        },
+        "b": {
+          "type": "boolean"
+        },
+        "c": {
+          "type": "integer"
+        },
+        "d": {
+          "type": "integer"
+        },
+        "e": {
+          "type": "string"
+        },
+        "f": {
+          "type": "string"
+        },
+        "foo": {
+          "$ref": "#/types/example::Foo"
+        },
+        "bar": {
+          "$ref": "#/types/example::Foo"
+        },
+        "baz": {
+          "type": "array",
+          "items": {
+            "$ref": "#/types/example::Foo"
+          }
+        }
+      },
+      "required": ["a", "c", "e"],
+      "inputProperties": {
+        "a": {
+          "type": "boolean"
+        },
+        "b": {
+          "type": "boolean"
+        },
+        "c": {
+          "type": "integer"
+        },
+        "d": {
+          "type": "integer"
+        },
+        "e": {
+          "type": "string"
+        },
+        "f": {
+          "type": "string"
+        },
+        "foo": {
+          "$ref": "#/types/example::Foo"
+        },
+        "bar": {
+          "$ref": "#/types/example::Foo"
+        },
+        "baz": {
+          "type": "array",
+          "items": {
+            "$ref": "#/types/example::Foo"
+          }
+        }
+      },
+      "requiredInputs": ["a", "c", "e"],
+      "plainInputs": ["a", "b", "c", "d", "e", "f", "bar", "baz"],
+      "type": "object"
+    }
+  },
+  "language": {
+    "csharp": {},
+    "go": {
+      "importBasePath": "github.com/pulumi/pulumi/pkg/v3/codegen/internal/test/testdata/simple-plain-schema/go/example",
```

Comment:
This path doesn't match. It should be
`github.com/pulumi/pulumi/pkg/v3/codegen/internal/test/testdata/simple-plain-schema-with-root-package/go/example`

Reply:
Fixed in b5ead72 - great catch!